index.js

```diff
@@ -2,13 +2,26 @@
 const { exec } = require('child_process');
 const path = require('path');
 
-const project_name = process.argv[2];
-const group_name = process.argv[3];
+let project_name = "";
+let group_name = "";
+
+if(process.argv.length > 4){
+  let refinedProcess = process.argv;
+  refinedProcess = refinedProcess.slice(2).join(' ');
+  project_name = refinedProcess.split(" -- ")[0];
+  group_name = refinedProcess.split(" -- ")[1];
+}else{
+  project_name = process.argv[2];
+  group_name = process.argv[4];
+}
+
+console.log("project_name", project_name);
```

Comment:
I believe that a console.log is indeed necessary, but I also believe that we could add other information to make it more comprehensive.

Reply:
New pull request created with more detailed logs!